### ISSUE TITLE
Define color scheme

### ISSUE THREAD

**Issue by emelinesen** [emelinesen/semester-project#7]
Settle on a maximum of two colors, the third should be neutral. 

- [ ] Primary
- [ ] Secondary
- [ ] Neutral

- [ ] Check contrast
